_inc/client/components/plans/plan-icon/index.jsx

```diff
@@ -146,6 +148,26 @@ export default class PlanIcon extends Component {
 		);
 	}
 
+	getSearchIcon() {
+		return (
+			<svg
+				className={ this.getIconClassNames( 'dops-plan-icon__search' ) }
+				xmlns="http://www.w3.org/2000/svg"
+				version="1.1"
+				x="0"
+				y="0"
+				viewBox="0 0 124 124"
```

Comment:
@gibrown it looks like changing the viewBox to "0 0 64 64" fixes your display issue

Reply:
oh look at that! thanks!